src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2018. The Hyve and respective contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * See the file LICENSE in the root of this repository.
+ *
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+import * as React from 'react';
+import DefaultTooltip from '../defaultTooltip/DefaultTooltip';
+import { observer } from 'mobx-react';
+import { JsonToTable } from 'react-json-to-table';
+import './StudyTagsTooltip.scss';
+import { remoteData } from '../../api/remoteData';
+import client from "shared/api/cbioportalClientInstance";
+import { MobxPromiseUnionType } from 'mobxpromise';
+import Loader from '../loadingIndicator/LoadingIndicator';
+
+export type StudyTagsTooltipProps = {
+    studyDescription: string;
+    studyId: string;
+    key: number;
+    mouseEnterDelay: number;
+    placement: string;
+    children: any;
+};
+
+@observer
+export default class StudyTagsTooltip extends React.Component<StudyTagsTooltipProps, {}> {
+
+    readonly studyMetadata = remoteData({
+        invoke: async () => {
+            return client.getTagsUsingGET({studyId: this.props.studyId});
+        },
+        onError: (error) => {
+            console.error("Error on getting study tags.", error);
+        }
+    });
+    
+    constructor(props: StudyTagsTooltipProps, context: any) {
+        super(props, context);
+    }
+
+    parseDescription(description:string) {
+        return {__html: description};
+    }
+
+    renderTooltip = (meta: MobxPromiseUnionType<any>) => {
+        let overlay:any = '';
+        if (meta.isPending) {
+            overlay = <Loader isLoading={true}/>;
+        }
+        if (meta.isComplete) {
+            const resultKeyLength = Object.keys(meta.result).length;
+            const description = <div dangerouslySetInnerHTML={this.parseDescription(this.props.studyDescription)}/>;
+            overlay = resultKeyLength > 0 ? ([description, <br/>, <JsonToTable json={meta.result}/>]) : description;
+        }
+        if (meta.isError) {
+            overlay = 'error';
+        }
+
+        return (<DefaultTooltip
```

Comment:
I believe that this will result in an <a> tag inside of an <a> tag (see outer component). I need to test it though.

Reply:
@alisman  Please, test. I've tried to remove the `{content}` from line 337 in `StudyList.tsx` (I think that can be the duplication) but then it fails to render the component.

Reply:
@alisman I moved the `<a>` so that it only includes the `{content}` part (in the outer component, as we discussed). So I think this is fixed.